huuduy216/Talkie-Web
# Fix margin issue on homepage

## Description: 

Change checking for class name injection on homepage for dev environment.

## To Test:

- On your machine, if you are using XAMPP, make sure all your Talkie-Web content is extracted directly to htdocs instead of inside a folder.
-  Run `composer update joseph280996/alwaysdisplaydiscussionlist` to put the new version into the `vendor` folder
- Turn on XAMPP and go to homepage, open inspector to see if `main` tag with class `App-content` has `mr-0` class injected to it.
